modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
```

Comment:
1. Should it say "cluster overview dashboard" or just "overview dashboard"?
2. Same question about the capitalization, just in case. <rimshot> 😂

Reply:
Using the [OCP docs standard](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) the title should be something like 'About the Overview Dashboard' (Pan's question about capitals still stands :)) 
The module id should match the title, and also *needs* to match the filename. The `cnv-` prefix is important.

Reply:
Thanks, made all changes. I am going with overview dashboard, using sentence casing as it seems this is how we refer to other UIs, as opposed to products which seem to be capped.